### PR TITLE
Minor Block javadoc improvements

### PR DESCRIPTION
**Pull request by Juuxel**
- Added mention of `AbstractBlock` as it contains many of the useful block methods
- "logic" is uncountable (and thus singular) in this context
- Fixed broken `{@link}` to a member
- Fixed broken grammar ("the possible states ... ~~is~~ *are*")

### CODE REVIEW

File: mappings/net/minecraft/block/Block.mapping [hunk 1]
```diff
@@ -1,6 +1,6 @@
 CLASS net/minecraft/class_2248 net/minecraft/block/Block
-	COMMENT A block is a voxel in a {@linkplain World world}.
-	COMMENT This class and its subclasses define all logics for those voxels.
+	COMMENT A block is a voxel in a {@linkplain World world}. {@link AbstractBlock},
+	COMMENT this class and its subclasses define all logic for those voxels.
```
Comment (haykam821) [inline mappings/net/minecraft/block/Block.mapping]:
```suggestion
	COMMENT this class, and its subclasses define all logic for those voxels.
```

Comment (Juuxel) [reply mappings/net/minecraft/block/Block.mapping]: It [works both with and without](https://en.wikipedia.org/wiki/Serial_comma) (unless you follow a specific style guide, which we don't).

Comment (Shnupbups) [reply mappings/net/minecraft/block/Block.mapping]: Personally I prefer the Oxford comma to be present

**Review by Shnupbups — APPROVED** [review]

File: mappings/net/minecraft/block/Block.mapping (hunk 1 above)
Comment (liach) [reply mappings/net/minecraft/block/Block.mapping]: think american english requires the comma before and unless it's only two items, as seen in americn college test

Comment (Juuxel) [reply mappings/net/minecraft/block/Block.mapping]: It's Oxfordised now.

**Review by haykam821 — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by enbrain — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**